### ISSUE TITLE
Layer Normalization Layer

### ISSUE THREAD

**Issue by vandenbergheluke** [vandenbergheluke/neuralwindow#122]
https://pytorch.org/docs/stable/generated/torch.nn.LayerNorm.html. Required for transformer implementation.
